src/FunFunFactory.php

```diff
@@ -576,7 +576,8 @@ public function newAddDonationUseCase(): AddDonationUseCase {
 				$this->newPaymentTypeValidator(),
 				$this->newBankDataValidator(),
 				$this->getMailValidator()
-			)
+			),
+			$this->newGeneralizedReferrer()
```

Comment:
This is a bad class name btw: adjective + noun. This would be better: `ReffererGeneralizer`